src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1079,6 +1081,24 @@ public static String prefixName(String name) {
     return PDAO_PREFIX + name;
   }
 
+  private Schema buildRowMetadataSchema() {
+    List<Field> fieldList = new ArrayList<>();
+    fieldList.add(Field.of(PDAO_ROW_ID_COLUMN, LegacySQLTypeName.STRING));
+    fieldList.add(
+        Field.newBuilder("ingested_by", LegacySQLTypeName.STRING)
+            .setMode(Field.Mode.REQUIRED)
```

Comment:
Why is this `REQUIRED` but `PDAO_ROW_ID_COLUMN` is `NULLABLE`? Is there a case where `PDAO_ROW_ID_COLUMN` will be `null`?

Reply:
Good catch - I've made the row id column required.